gammapy/data/metadata.py

```diff
@@ -162,3 +162,135 @@ def from_header(cls, events_hdr, format="gadf"):
         kwargs["optional"] = optional
 
         return cls(**kwargs)
+
+
+class EventListMetaData(MetaData):
```

Comment:
I do not see the point of such redundancy with the `ObservationMetaData`. Why not including it in this class?